src/main/webapp/app/exercises/shared/exercise-scores/exercise-scores.component.ts

```diff
@@ -42,6 +43,7 @@ enum FilterProp {
     selector: 'jhi-exercise-scores',
     styleUrls: ['./exercise-scores.component.scss'],
     templateUrl: './exercise-scores.component.html',
+    providers: [ExerciseCacheService],
```

Comment:
We enable caching on a component basis by adding the cache service to the providers